hardware/raspberrypi/datasheets.md

```diff
@@ -0,0 +1,18 @@
+## Datasheets
+
+### SoC's
+
+The following datasheets are available for the SoC (System On Chip - the main processing chip) devices used on the Raspberry Pi platform.
+
+[BCM2835](bcm2835/BCM2835-ARM-Peripherals.pdf) - The SoC used on the Raspberry Pi 0 and 1. This datasheet is also applicable to the Raspberry Pi 2 and 3, with the BCM2836 and BC2837 SoC's respectively, as the Videocore GPU is the same.
```

Comment:
This feels a little bit clumsy? Perhaps:
The SoC used on the Raspberry Pi 0 and 1. This datasheet is also applicable to the BCM2836 and BC2837 SoCs (used on the Pi2 and Pi3 respectively), as they use the same VideoCore IV GPU.

:question:

Reply:
...or perhaps "Large parts of this datasheet are also applicable to the BCM2836 and BC2837 SoCs ..." ?

Reply:
Yes, much better. Ta.